var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -50,12 +63,81 @@ class Mesa(AutotoolsPackage):
     depends_on('libxext')
     depends_on('libxdamage')
     depends_on('libxfixes')
+    depends_on('expat')
 
     depends_on('glproto@1.4.14:', type='build')
     depends_on('dri2proto@2.6:', type='build')
     depends_on('dri3proto@1.0:', type='build')
     depends_on('presentproto@1.0:', type='build')
     depends_on('pkg-config@0.9.0:', type='build')
+    depends_on('binutils', type='build')
```

Comment:
We generally avoid adding binutils dependencies because technically every package in Spack depends on ld, much like every package depends on curl to fetch it. It's more of a Spack dependency than a Mesa dependency.